taskfile/cache.go

```diff
@@ -2,24 +2,51 @@ package taskfile
 
 import (
 	"crypto/sha256"
+	"errors"
 	"fmt"
 	"os"
 	"path/filepath"
 	"strings"
+	"time"
 )
 
+// DefaultCacheTTL is one day (24 hours).
```

Comment:
```suggestion
// DefaultCacheTTL is used when WitTTL is not used
```

There is no need add a comment that could be outdated next time the value is changed. I suggest a better comment

Reply:
741743b7137328a81c95ff33b82f0bd02ebb581f will resolve.